ca_on_toronto/bills.py

```diff
@@ -32,9 +32,56 @@
     'Without Recs': None,
     'Waive Referral': None,
     # Made this one up
-    'Introduced': 'introduction',
+    'Filed': 'filing',
+    # Motion actions
+    'Add New Business at Committee': 'introduction',
+    'Adopt Item as Amended': 'passage',
+    'Adopt Item': 'passage',
+    'Adopt Minutes': 'passage',
+    'Adopt Order Paper as Amended': 'passage',
+    'Adopt Order Paper': 'passage',
+    'Amend Item (Additional)': 'amendment-passage',
+    'Amend Item': 'amendment-passage',
+    'Amend Motion': None,
+    'Amend the Order Paper': None,
+    'Confirm Order': None,
+    'Defer Item Indefinitely': 'deferred',
+    'Defer Item': 'deferred',
+    'End Debate': None,
+    'Extend the Meeting': None,
+    'Introduce Motion without Notice': 'introduction',
+    'Introduce Report': None,
+    'Introduce and Pass Confirmatory Bill': None,
+    'Introduce and Pass General Bills': None,
+    'Meet in Closed Session': None,
+    'Re-open Item': None,
+    'Receive Item': None,
+    'Reconsider Item': None,
+    'Reconsider Vote': None,
+    'Refer Item': 'committee-referral',
+    'Remove from Committee': None,
+    'Waive Notice': None,
+    'Waive Referral': None,
+    'Withdraw a Motion': None,
+    'Withdraw an Item': 'withdrawal',
+}
+
+RESULT_MAP = {
+    'Carried': 'pass',
+    'Lost': 'fail',
+    'Lost (tie)': 'fail',
+    'Redundant': 'fail',
+    'Final': 'fail',
```

Comment:
I've only seen it once, and it's on my TODO list to email the clerk's office, but something about quorum being lost, it seems:
http://app.toronto.ca/tmmis/viewAgendaItemHistory.do?item=2016.SC13.20